inginious/frontend/pages/course_admin/submissions.py

```diff
@@ -58,7 +58,8 @@ def page(self, course, msgs=None):
         msgs = msgs if msgs else []
         
         user_input = self.get_input()
-        data, classroom = self.get_submissions(course, user_input)  # ONLY classrooms user wants to query
+        data, classroom = self.get_submissions(course, user_input)
+        data_table, classroom = self.get_submissions(course, user_input,True)# ONLY classrooms user wants to query
```

Comment:
1. Why it is necessary to call the `get_submissions` method twice? This means that not only the inneficient query is still done, but another one.
2. Check uses of `data` and `data_table` in this method. Shouldn't be consistent?